expat/lib/xmlparse.c

```diff
@@ -2080,7 +2080,7 @@ XML_GetBuffer(XML_Parser parser, int len)
       parser->m_bufferLim = newBuf + bufferSize;
 #ifdef XML_CONTEXT_BYTES
       if (parser->m_bufferPtr) {
-        int keep = (int)EXPAT_SAFE_PTR_DIFF(parser->m_bufferPtr, parser->m_buffer);
+        keep = (int)EXPAT_SAFE_PTR_DIFF(parser->m_bufferPtr, parser->m_buffer);
```

Comment:
It turns out the the whole line is a re-calculation of the same value and hence can be dropped altogether.

Reply:
What do you think?